libr/core/cmd_open.c

```diff
@@ -693,22 +693,19 @@ static int cmd_open(void *data, const char *input) {
 		}
 		break;
 	case 'c':
-		if ('?' == input[1]) {
-			const char *help_msg[] = {
-				"oc"," [file]","open core file, like relaunching r2",NULL
-			};
-			r_core_cmd_help (core, help_msg);
-			break;
-		}
-		if (r_sandbox_enable (0)) {
-			eprintf ("This command is disabled in sandbox mode\n");
-			return 0;
-		}
-		// memleak? lose all settings wtf
-		// if load fails does not fallbacks to previous file
-		r_core_fini (core);
-		r_core_init (core);
 		if (input[1] && input[2]) {
+			if (!r_file_exists (input + 2)) {
```

Comment:
thats not correct. files in r2 can be anywhere, in remote places, or even virtual ones hidden under URI handlers